src/components/ArticleShowCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useEffect } from "react";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+import NavBar from "./NavBar";
+
+export default function ArticleShowCard({ articleList, setArticleList }) {
+  const { article_id } = useParams();
+
+  const getArticleById = () => {
+    let path = `https://nc-news-back-end-project.onrender.com/api/articles/${article_id}`;
+    return axios
+      .get(path)
+      .then((response) => {
+        return response.data;
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+  useEffect(() => {
+    getArticleById().then((response) => {
```

Comment:
I'd recommend making this function take an argument for the article_id to be passed into the string, even though here it can access the id in scope, the function is more reusable in other places if it can be passed in. Otherwise outside of this scope the function won't be able to make a request